src/components/animation/_tests/ParallelAnimationFactory.test.brs

```diff
@@ -0,0 +1,94 @@
+' @import /components/KopytkoTestSuite.brs from @dazn/kopytko-unit-testing-framework
+' @import /components/rokuComponents/Animation.brs
+' @mock /components/animator/AnimatorFactory.brs
+function TestSuite__ParallelAnimationFactory() as Object
+  ts = KopytkoTestSuite()
+  ts.name = "ParallelAnimatorFactory"
+
+  ts.setBeforeEach(sub (ts as Object)
+    m.__parallelAnimationFactory = ParallelAnimationFactory()
+
+    m.__mocks.animatorFactory = {
+      returnValue: Animation(),
+    }
+  end sub)
+
+  ts.addTest("it creates animation object with given config", function (ts as Object) as String
+    ' Given
+    options = {
+      delay: Csng(0.2),
+      repeat: true,
+      animations: {
+        elementOne: {
+          delay: Csng(0.2),
+          duration: Csng(20),
+          easeFunction: "linear",
+          easeInPercent: 0.1,
+          easeOutPercent: 0.2,
+          optional: false,
+          repeat: false,
```

Comment:
I don't understand, aren't `fields` necessary? Otherwise, it makes no sense to me